core/src/main/scala/org/apache/spark/util/JsonProtocol.scala

```diff
@@ -469,6 +471,7 @@ private[spark] object JsonProtocol {
 
   def jobStartFromJson(json: JValue): SparkListenerJobStart = {
     val jobId = (json \ "Job ID").extract[Int]
+    val submissionTime = (json \ "Submission Time").extractOpt[Long]
```

Comment:
Similarly, you should also add a backwards compatibility test; this can be a few lines in the existing "SparkListenerJobStart backward compatibility" test: https://github.com/sarutak/spark/blob/SPARK-5231/core/src/test/scala/org/apache/spark/util/JsonProtocolSuite.scala#L240